cli/tests/integration/watcher_tests.rs

```diff
@@ -496,15 +505,21 @@ fn run_watch() {
     .unwrap();
   let (mut stdout_lines, mut stderr_lines) = child_lines(&mut child);
 
-  assert_contains!(stdout_lines.next().unwrap(), "Hello world");
```

Comment:
@bartlomieju I think to make the other tests not flaky we will have to switch them to use `wait_for`/`wait_contains` as well.